pages/docs/vi/vi-headless.md

```diff
@@ -0,0 +1,91 @@
+# First Steps
+
+## Headless Set up
+&nbsp;
+Don’t have an extra keyboard or HDMI cable? Here’s how to do a headless [Treehouses](http://download.treehouses.io/) install on your Pi.
+&nbsp;
+Here is the list of hardware requirements once again for you to refer to.
+&nbsp;
+#### Hardware requirement review:
+1.  A Raspberry Pi 4
+1.  A power cord for the Pi
+	- Standard micro USB power cord: 2.4A for Pi 3 and Pi 3b+, 1.2A for Pi Zero W, or 15.3W for Pi 4
+1.  3 Class 10 microSD cards
+	- minimal 16GB, but we recommend 32GB or greater
+1.  A microSD card reader
+	- Check your computer if you have a slot for a card reader otherwise use a USB reader
+1.  Ethernet cable or wifi dongle
+1.  USB keyboard, monitor, and HDMI cable
+	- Optional if using your own laptop via VNC
+1.  8 GB of hard-drive space
+1.  8 GB of memory (RAM)
+
+
+
+## Step 0 - Gather Items
+
+You just flashed the latest treehouses image on your SD card and now you want to get it running. 
+
+Gather your Pi, the SD card, an ethernet cable, your laptop, and the Pi's power cord.
+
+## Step 1 - SSH file
+
+Place your SD card into your laptop and open the boot file.
+
+Enable [SSH](https://en.wikipedia.org/wiki/Secure_Shell) by placing a file named “ssh” (without any extension) onto the boot partition of the SD card
+
+On mac: 
+1. Open textedit file
+2. make plain text
+3. select “save” and
+4. deselect “if no ext. is provided, use .txt”
+5. rename “ssh” and save inside boot file
+
+## Step 2: Get your RPi Up and Running
+
+Unmount your SD card and place it inside the RPi. 
+Now connect the Pi to your laptop via ethernet cable and plug in it's power cord. 
+
+Give it a few minutes to get started and check for the flashing green light to make sure the image burned correctly. 
+
+## Step 3: Accessing Treehouses 
+
+Now you will open up your terminal or command prompt. To configure your Pi, you will need the IP address. To do this, enter the command:
+~~~
+ping treehouses.local
+~~~
+Once the IP address appears press Ctrl + C.
```

Comment:
should include example output and other methods of finding pi; if lazy you can just link to [Raspberry Pi documentation](https://www.raspberrypi.org/documentation/remote-access/ip-address.md) or our own first steps finding pi page

Reply:
The details can just be copied from my [Pihole blog post](https://treehouses.io/#!./pages/blog/20190717-pihole.md)